### DISCUSSION TITLE
clkitemでUWSCではクリックできたがUWSCRではクリックできない

### DISCUSSION THREAD

**Discussion by stuncloud** [stuncloud/UWSCR#62] [Q&A]
UWSCでclkitemを使ってクリックできていたものがUWSCRではクリックできない、ということがあったら情報をいただきたいです

- 対象ウィンドウ (アプリケーション情報も)
- 対象アイテム (ボタンなど)
- 指定する`CLK定数` の組み合わせ

等を教えてください

**Comment by DIYJii** [comment]
1. UWSCのHelp画面で2つ見つけました
  　左側のリストボックスでキーワードを表示。
　　a.表示されているキーワード：アイテム名は表示されているキーワードどれでも
　　b.リストボックスのスライダー：アイテム名は「表示位置」
　　　CLK_MUSMOVEを指定してもカーサもそこに移動はしません。

2. もう一つ別に、 ダブル・クリックの問題
　　UWSCで　CLK_ACC を指定しないで、DeskTop上のアイコン
　　又は上述のUWSC Help画面のキーワードにCLKITEMを掛けると、
　　自動的にダブル・クリックをしてアプリの立ち上げ又は説明画面　
　　に移動までまでやってくれます。
　　UWSCRでは一回のクリックだけなので、それらは起きません。
       この件は、ただ、違いをレポートしているだけで、どうしろと言っている訳では、ありません。
